### DISCUSSION TITLE
1.2 - long overdue edition

### DISCUSSION THREAD

**Discussion by SeenKid** [SeenKid/flipper-zero-bad-usb#5] [Announcements]
📌 Fixed the Open URL script on Iphones ( Thanks to yellowstar520 🌷)
📌 Added a few scripts 

⚠️ Your browser may not let you download the repo because of Memz ⚠️
( https://github.com/SeenKid/flipper-zero-bad-usb/tree/main/utils/malware/memz ) 
💥 Which is a harmless executable. You can bypass that by downloading the file with Firefox

📈Some stats 📊
✅3.7K downloads ⬇️
✅226 stars ⭐
✅71700 views on the repo 👀

 Thank you everyone ! 💖

<hr /><em>This discussion was created from the release <a href='https://github.com/SeenKid/flipper-zero-bad-usb/releases/tag/'>1.2 - long overdue edition</a>.</em>

**Comment by iDefalt666** [comment]
Hi, i tried downloading it through FF, but keeps saying virus detected. What file is detect as the virus?

**Reply by sun12yyds** [reply]
i like it____
 memz

**Reply by SeenKid** [reply]
Memz indeed, guess I should remove it. Try to disable your AV / Windows Defender

**Reply by SeenKid** [reply]
Download this :
![image](https://github.com/user-attachments/assets/bd74fa75-8528-4018-b540-6c18bdf461f1)

**Comment by sun12yyds** [comment]
yes,its memz
________________________________
发件人: ***@***.*** ***@***.***>
发送时间: 2024年7月29日 14:24
收件人: SeenKid/flipper-zero-bad-usb ***@***.***>
抄送: sun12yyds ***@***.***>; Comment ***@***.***>
主题: Re: [SeenKid/flipper-zero-bad-usb] 1.2 - long overdue edition (Discussion #5)


Download this :
image.png (view on web)<https://github.com/user-attachments/assets/bd74fa75-8528-4018-b540-6c18bdf461f1>

―
